sdk/rust/crates/dagger-sdk/src/core/engine.rs

```diff
@@ -30,6 +31,10 @@ impl Engine {
             return Ok((conn, None));
         }
 
+        if let Ok((conn, child)) = self.from_local_cli(cfg).await {
+            return Ok((conn, Some(child)));
+        }
```

Comment:
If we put `?` after await then it would raise with error:

```
Error: failed to connect to dagger engine

Caused by:
    environment variable not found

Location:
    crates/dagger-sdk/examples/first-pipeline/main.rs:3:18
```

The error above come from running `cargo run --example first-pipeline`. It probably because of I used `await?` during fetch env variable. What should I suppose to fix in this case?

Reply:
You can just return a None instead then.

We don't care about the error, so we just return an Option, 

If you don't need the error at all.

It can also be done like so.

```rust
if let Some((conn, child)) = self.from_local_cli(cfg).await.ok() { //-> this will transform an Err into an Option. 
```